policies/opnrvn-p-2.yaml

```diff
@@ -6,9 +6,19 @@ description: >
   Google Cloud Security Best Practices analyzes accounts for security best practices and is modelled on guidance published by the Center for Information Security.
 enabled: true
 rules:
```

Comment:
Rules 58-59-60 relate to AWS and are already mentioned in p-1 script.
Removed from here